### PR TITLE
Wslink: use wslink for communication, replacing autobahn wamp

### PR DESCRIPTION
**Pull request by aronhelser**
Depends on server changes in ParaView, https://gitlab.kitware.com/paraview/paraview/merge_requests/1630

and changes in PVW: need a new version after this merges: https://github.com/Kitware/paraviewweb/pull/372

### CODE REVIEW

**Comment by aronhelser** [comment]
ParaView change is merged. There's a fix in VTK which will be picked up by https://gitlab.kitware.com/paraview/paraview/merge_requests/1800

Then we need the PVW release from https://github.com/Kitware/paraviewweb/pull/372

**Comment by aronhelser** [comment]
Should be final, now, but depends on two in-progress MRs, [Paraview MR](https://gitlab.kitware.com/paraview/paraview/merge_requests/1835) [VTK MR](https://gitlab.kitware.com/vtk/vtk/merge_requests/3117)

**Comment by aronhelser** [comment]
Both VTK and Paraview MRs are in.
@jourdain want to take a look?

File: src/network.js [hunk 1]
```diff
@@ -45,7 +46,7 @@ function exit(timeout = 60) {
 }
 
 function connect(config = {}) {
-  smartConnect = new SmartConnect(config);
+  smartConnect = SmartConnect.newInstance({ config });
```
Comment (jourdain) [inline src/network.js]: Looks like a typo `{{ ?? }}`?

**Comment by jourdain** [comment]
LGTM, you can merge if you want.

**Pull request merged**